contracts/config/networks.json

```diff
@@ -1,6 +1,6 @@
 {
     "localGeth": {
-      "url": "http://127.0.0.1:17000",
```

Comment:
did you mean to commit this port change?

Reply:
The port has changed after the geth upgrade. Since we don't use this networks.json in automated change it wasn't caught.